src/awx_plugins/credentials/azure_kv.py

```diff
@@ -1,63 +1,81 @@
-from azure.keyvault.secrets import SecretClient
 from azure.identity import ClientSecretCredential
+from azure.keyvault.secrets import SecretClient
 from msrestazure import azure_cloud
 
-from .plugin import CredentialPlugin
-
-from .plugin import translate_function as _
+from .plugin import CredentialPlugin, translate_function as _
 
 
 # https://github.com/Azure/msrestazure-for-python/blob/master/msrestazure/azure_cloud.py
-clouds = [vars(azure_cloud)[n] for n in dir(azure_cloud) if n.startswith("AZURE_") and n.endswith("_CLOUD")]
-default_cloud = vars(azure_cloud)["AZURE_PUBLIC_CLOUD"]
-
-
-azure_keyvault_inputs = {
-    'fields': [
-        {
-            'id': 'url',
-            'label': _('Vault URL (DNS Name)'),
-            'type': 'string',
-            'format': 'url',
-        },
-        {'id': 'client', 'label': _('Client ID'), 'type': 'string'},
-        {
-            'id': 'secret',
-            'label': _('Client Secret'),
-            'type': 'string',
-            'secret': True,
-        },
-        {'id': 'tenant', 'label': _('Tenant ID'), 'type': 'string'},
-        {
-            'id': 'cloud_name',
-            'label': _('Cloud Environment'),
-            'help_text': _('Specify which azure cloud environment to use.'),
-            'choices': list(set([default_cloud.name] + [c.name for c in clouds])),
-            'default': default_cloud.name,
-        },
-    ],
-    'metadata': [
-        {
-            'id': 'secret_field',
-            'label': _('Secret Name'),
-            'type': 'string',
-            'help_text': _('The name of the secret to look up.'),
-        },
-        {
-            'id': 'secret_version',
-            'label': _('Secret Version'),
-            'type': 'string',
-            'help_text': _('Used to specify a specific secret version (if left empty, the latest version will be used).'),
-        },
-    ],
-    'required': ['url', 'client', 'secret', 'tenant', 'secret_field'],
-}
+clouds = [
+    vars(azure_cloud)[n] for n in dir(azure_cloud)
+    if n.startswith('AZURE_') and n.endswith('_CLOUD')
+]
+default_cloud = vars(azure_cloud)['AZURE_PUBLIC_CLOUD']
+
+
+azure_keyvault_inputs = {'fields': [{'id': 'url',
```

Comment:
Weird that it made this edit. It's probably because of mixed style inside (perhaps those inline dicts). It should be possible to dedent it if you want.

Reply:
Looks like long unsplittable lines were also causing this.

I've made https://github.com/ansible/awx-plugins/pull/20 to show how the diff may be smaller.

Also, this PR still triggers formatting (https://results.pre-commit.ci/run/github/836873755/1724796257.Vt0Lwn2FSQ-J0oVorFr3bw) while mine does not — it needs a couple of reruns to settle, between several tools changing files.